README.md

```diff
@@ -50,6 +50,13 @@ A spring bean:
 
    }
 ``` 
+##Configuration for Service Authentication filter
+The following values must be provided to enable a ServiceAuthFilter bean:
```

Comment:
What does the filter do? (I know what It does but that’s missing here)

Reply:
ok get it, I have added some more information.